src/hooks/useSwapDerivedOutputs.ts

```diff
@@ -111,6 +111,7 @@ const getInputAmount = async (
 
     const rand = Math.floor(Math.random() * 100);
     Logger.debug('Getting quote ', rand, { quoteParams });
+    // @ts-ignore About to get quote
```

Comment:
can we add a lil comment that says `do not deleeeet` on these

Reply:
no